homeassistant/components/entur_public_transport/manifest.json

```diff
@@ -1,10 +1,12 @@
 {
   "domain": "entur_public_transport",
   "name": "Entur",
-  "codeowners": ["@hfurubotten"],
+  "codeowners": ["@hfurubotten", "@SanderBlom"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/entur_public_transport",
+  "integration_type": "service",
   "iot_class": "cloud_polling",
   "loggers": ["enturclient"],
-  "quality_scale": "legacy",
+  "quality_scale": "bronze",
```

Comment:
Please change the quality scale level and record in a separate PR from other changes.